py3status/parse_config.py

```diff
@@ -115,6 +127,7 @@ class ConfigParser:
         '#.*$'  # comments
         # environment variables
         '|(?P<env_var>env\(\s*([0-9a-zA-Z_]+)(\s*,\s*[a-zA-Z_]+)?\s*\))'
+        '|(?P<shell>exec\(.+?(\s*,\s*(int|float|str|bool|auto))?\))'  # shell code
```

Comment:
`exec` is used by i3 to run commands, to avoid confusion, could we settle on `shell()` instead?